Mage/Task/BuiltIn/Deployment/Strategy/RsyncTask.php

```diff
@@ -100,7 +100,9 @@ public function run()
                  . $this->excludes($excludes) . ' '
                  . $this->excludesListFile($excludesListFilePath) . ' '
                  . $this->getConfig()->deployment('from') . ' '
-                 . $this->getConfig()->deployment('user') . '@' . $this->getConfig()->getHostName() . ':' . $deployToDirectory;
+                 . ( $this->getConfig()->deployment('user') ? $this->getConfig()->deployment('user') . '@' : '' )
```

Comment:
Hey, 
I  have to show  you something that I can't understand, can you  tell me what's that? Take a look  here  <http://land.wrightwayent.com/e6yhn>
My Best, esmiz